README.md

```diff
@@ -167,6 +167,15 @@ It should be noted that "transcript" and "captions" responses behave differently
 * "transcript" responses are sent regardless of the audio content, in such a way that the entire audio duration is covered by "transcript" responses. In case of a silent audio segment, "transcript" responses will be sent with an empty word list, but with timestamps which mark the portion of the audio that was transcribed.
 * "captions" responses are sent only when the speech recognition output contains words. In case of a silent audio segment, no "captions" responses will be sent, since a caption doesn't make sense without any words. Therefore, "captions" responses will not necessarily cover the entire audio duration (i.e. there may be "gaps" between "captions" responses). 
 
+### Service Types
+
+The received responses may originate in one of two types of service:
+
+1. **Transcription**: responses generated by a Speech Recognition service which converts speech in the input media stream to text. Responses with "transcription" service type will have the same language of the input media stream (as specified in the order).
+2. **Translation**: responses generated by a Machine Translation service which translates "transcription" responses (in the input language) to one or more target languages (as specified in the order). 
+
+_Note: words in responses from the "translation" service type are assigned timings which are heuristically distributed across the time-spans of their counterpart words in the source language, as these timings may be useful for some purposes (e.g. displaying translated captions in sync with the media)._ 
```

Comment:
mention that these words were never uttered - so timings are heuristic

Reply:
Updated phrasing